cmd/drone-bazelisk-ecr/plugin.go

```diff
@@ -24,6 +24,9 @@ type plugin struct {
 	AccessKey        string `split_words:"true"`
 	SecretKey        string `split_words:"true"`
 	Bazelrc          string
+	Command          string
+	CommandArgs      string
+	TargetArgs       string
```

Comment:
It's unclear to me what `TargetArgs` would be used for based on the [CLI docs](https://bazel.build/reference/command-line-reference). It seems like this is probably redundant with the existing `Target` option.

Reply:
Target args are arguments passed to the thing being built rather than to the build system itself.

Reply:
Ah, that makes sense, thanks for clarifying.